src/Symfony/Bridge/Doctrine/ManagerRegistry.php

```diff
@@ -49,7 +49,7 @@ protected function resetService($name)
         }
         $manager->setProxyInitializer(\Closure::bind(
             function (&$wrappedInstance, LazyLoadingInterface $manager) use ($name) {
-                if (isset($this->aliases[$name = strtolower($name)])) {
+                if (isset($this->aliases[$name])) {
                     $name = $this->aliases[$name];
                 }
                 $method = !isset($this->methodMap[$name]) ? 'get'.strtr($name, $this->underscoreMap).'Service' : $this->methodMap[$name];
```

Comment:
@stof not sure here.. it seems forgotten? This is the only use-case of `Container::$underscoreMap` left.